docs/pages/clients/getting-started.md

```diff
@@ -51,9 +52,226 @@ When you first launch your application, you will see a screen that looks like th
 
 <ImageSpotlight alt="The launcher screen of the Development Client" src="/static/images/dev-client-launcher.png" style={{ maxWidth: 225}} />
 
-If a bundler is available on your local network, or you've signed in to your Expo account, you'll can connect to it directly from this screen.
+If a bundler is available on your local network, or you've signed in to your Expo account, you can connect to it directly from this screen.
 Otherwise, you can connect by scanning the QR code displayed by Expo CLI.
 
+## Customizing your runtime
+
+In the Expo Go client, you can already convert text to audio with [expo-speech](/versions/latest/sdk/speech), but what if you want to go the other direction and convert audio to text?  The community module [`@react-native-voice/voice`](https://github.com/react-native-voice/voice) provides this capability, and thanks to config plugins, you can add it into our project!
+
+Add the module to our project by installing the module
+
+<InstallSection packageName="expo-dev-client" cmd={["yarn add @react-native-voice/voice"]} hideBareInstructions />
+
+and registering the plugin in your app.json
+<!-- prettier-ignore -->
+```js
+"expo": {
+  "plugins": ["@react-native-voice/voice"]
+}
+```
+
+> ⚠️ Because adding this module changes our native runtime, you need to generate a new build of our development client with EAS before you'll be able to use it.
```

Comment:
```suggestion
> ⚠️ Because adding this module changes your native runtime, you need to generate a new development client build with EAS before you'll be able to use it, otherwise you'll get a "Native module cannot be null" error.
```